app/kumactl/data/install/k8s/tracing/jaeger/all-in-one-template.yaml

```diff
@@ -62,6 +62,10 @@ items:
                     protocol: TCP
                   - containerPort: 9411
                     protocol: TCP
+                  - containerPort: 4317
```

Comment:
Looks like maybe we need to set `COLLECTOR_OTLP_ENABLED=true`? https://www.jaegertracing.io/docs/1.42/getting-started/#all-in-one

Reply:
oh and we need to bump the jaeger version too.